packages/component/src/ErrorBoundary.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+class ErrorBoundary extends React.Component {
+  // Currently, all errors caught are fatal errors.
+  // We will bubble up the error and let React unmount us.
+  componentDidCatch(error) {
```

Comment:
Disappoint there isn't a hook for `componentDidCatch` yet

Reply:
Me too, looked through their docs 3 times and couldn't find any.